src/main/resources/uk/ac/ebi/ena/sra/schema/EGA.dac.xsd

```diff
@@ -10,7 +10,7 @@
   ~ specific language governing permissions and limitations under the License.
   -->
 
-<!-- version:1.12.0 -->
+<!-- version:1.13.0 -->
```

Comment:
Could we remove versions from the XSDs as we don't officially version them:
```
<!-- version:1.13.0 -->
```

Reply:
Done.

Reply:
Hi all, sorry to budge in, but apparently the [ReadTheDocs XSDs](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/programmatic.html#types-of-xml) still contain the version numbers (``v1.12.0``).